### PR TITLE
documentation: some minor clean up

### PR DESCRIPTION
**Pull request by mingboiz**
# What does this PR do?
This cleans up some minor documentation changes unrelated to debertav2 in the PR #15529  so I'm opening up this PR just for repo history. Please do let me know if this is alright 😄 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
https://github.com/huggingface/transformers/pull/15529
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: utils/documentation_tests.txt [hunk 1]
```diff
@@ -55,3 +55,4 @@ src/transformers/models/wav2vec2/modeling_wav2vec2.py
 src/transformers/models/wav2vec2/tokenization_wav2vec2.py
 src/transformers/models/wav2vec2_with_lm/processing_wav2vec2_with_lm.py
 src/transformers/models/wavlm/modeling_wavlm.py
+src/transformers/models/ctrl/modeling_ctrl.py
```
Comment (LysandreJik) [inline utils/documentation_tests.txt:58]: Is this an intended change?

**Review by LysandreJik — APPROVED** [review]
LGTM, thank you @mingboiz!

File: utils/documentation_tests.txt (hunk 1 above)
Comment (mingboiz) [reply utils/documentation_tests.txt:58]: Yes - during the [final merge](https://github.com/huggingface/transformers/pull/15529/files) this line was removed from `utils/documentation_tests.txt`. 

I wasn't able to track which exact commit in the linked PR that caused this change but I guess it happened because this line was added originally in one of the multiple rebases on main needed to pass CI (the PR was originally started in `4.17.dev0` 😅 ), so it was tracked as an additional but unrelated to debertav2 changes.

Restoring it for parity!

File: docs/source/en/main_classes/tokenizer.mdx [hunk 2]
```diff
@@ -18,9 +18,7 @@ Rust library [🤗 Tokenizers](https://github.com/huggingface/tokenizers). The "
 
 1. a significant speed-up in particular when doing batched tokenization and
 2. additional methods to map between the original string (character and words) and the token space (e.g. getting the
-   index of the token comprising a given character or the span of characters corresponding to a given token). Currently
-   no "Fast" implementation is available for the SentencePiece-based tokenizers (for T5, ALBERT, CamemBERT, XLM-RoBERTa
-   and XLNet models).
+   index of the token comprising a given character or the span of characters corresponding to a given token).
```
Comment (mingboiz) [inline docs/source/en/main_classes/tokenizer.mdx:21]: This was also an unrelated change to debertav2 in the linked PR - currently there's Fast implementation for the tokenizers of the mentioned models, so making the change here!

File: utils/documentation_tests.txt (hunk 1 above)
Comment (LysandreJik) [reply utils/documentation_tests.txt:58]: Indeed!

**Comment by LysandreJik** [comment]
Thank you!
